packages/atlas/src/components/FilterButton/FilterButton.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC, ReactNode, useRef, useState } from 'react'
+
+import { Counter } from '@/components/FilterButton/FilterButton.styles'
+import { Button } from '@/components/_buttons/Button'
+import { CheckboxProps } from '@/components/_inputs/Checkbox'
+import { CheckboxGroup } from '@/components/_inputs/CheckboxGroup'
+import { DialogPopover } from '@/components/_overlays/DialogPopover'
+
+export type FilterButtonProps = {
+  options: CheckboxProps[]
+  selected?: number[]
+  onApply: (ids: number[]) => void
+  label?: string
+  icon?: ReactNode
+}
+
+export const FilterButton: FC<FilterButtonProps> = ({ label, icon, options, onApply, selected = [] }) => {
```

Comment:
shouldn't we apply this new component to current filters? For instance in NftsView. If not could you create issue to add them to our views?